include/mbedtls/x509.h

```diff
@@ -294,7 +294,8 @@ typedef struct mbedtls_x509_subject_alternative_name {
     int type;                              /**< The SAN type, value of MBEDTLS_X509_SAN_XXX. */
     union {
         mbedtls_x509_san_other_name other_name; /**< The otherName supported type. */
-        mbedtls_x509_buf   unstructured_name; /**< The buffer for the un constructed types. Only dnsName currently supported */
+        mbedtls_x509_name directory_name;
+        mbedtls_x509_buf unstructured_name; /**< The buffer for the un constructed types. Only dnsName currently supported */
```

Comment:
Why are we adding this here? This is part of https://github.com/Mbed-TLS/mbedtls/pull/7025.

Reply:
What do you mean? Fixed whitespace?

Reply:
Oh sorry. It was too early. `unstructured_name` was already here and `directory_name` is inserted before.